terraform/cluster/aws/main.tf

```diff
@@ -10,9 +10,10 @@ provider "kubernetes" {
 }
 
 locals {
-  availability_zones     = var.availability_zones != "" ? compact(split(",", var.availability_zones)) : data.aws_availability_zones.available.names
-  network_resource_count = var.high_availability ? 3 : 2
-  oidc_sub               = "${replace(aws_iam_openid_connect_provider.cluster.url, "https://", "")}:sub"
+  availability_zones       = var.availability_zones != "" ? compact(split(",", var.availability_zones)) : data.aws_availability_zones.available.names
+  network_resource_count   = var.high_availability ? 3 : 2
+  oidc_sub                 = "${replace(aws_iam_openid_connect_provider.cluster.url, "https://", "")}:sub"
+  gpu_tag_disabled_regions = ["eu-north-1"]
```

Comment:
I wonder if we can find a list of not supported regions in aws website and add them all in this PR

Reply:
Yeah I thought that too, but didn't find any documentation for that